allennlp/commands/evaluate.py

```diff
@@ -118,13 +119,113 @@ def add_subparser(self, parser: argparse._SubParsersAction) -> argparse.Argument
             help="outputs tqdm status on separate lines and slows tqdm refresh rate",
         )
 
+        subparser.add_argument(
+            "--auto-names",
+            default="NONE",
+            help="Automatically create output names for each evaluation file.",
```

Comment:
Can this be clarified further? Will `METRICS` only have metrics in the file, and `PREDS` will have only predictions?

Reply:
I updated the docs to better reflect that. Metrics will only overwrite the name of the output file if `auto_names=="METRICS" or auto_names=="ALL"`. The same is true for the predictions output file but `"PREDS"` instead of `"METRICS"`.